bdsx/event_impl/entityevent.ts

```diff
@@ -418,15 +422,49 @@ function onPlayerAttack(player:Player, victim:Actor, cause:Wrapper<ActorDamageCa
 }
 const _onPlayerAttack = procHacker.hooking("Player::attack", bool_t, null, Player, Actor, Wrapper.make(int32_t))(onPlayerAttack);
 
+events.packetBefore(MinecraftPacketIds.InventoryTransaction).on((pk, ni) => {
+    if (pk.transaction.type === ComplexInventoryTransaction.Type.NormalTransaction) {
+        const transaction = pk.transaction.data;
+        const src = InventorySource.create(ContainerId.Inventory, InventorySourceType.ContainerInventory);
+        const actions = transaction.getActions(src);
+        if (actions.length === 1) {
+            const player = ni.getActor()!;
+            const itemStack = player.getInventory().getItem(actions[0].slot, ContainerId.Inventory);
+            src.destruct();
+            const event = new PlayerDropItemEvent(player, itemStack, false, actions[0].slot);
+            const canceled = events.playerDropItem.fire(event) === CANCEL;
+            _tickCallback();
+            decay(itemStack);
+            if (canceled) {
+                ni.getActor()!.sendInventory();
+                return CANCEL;
+            }
+        }
+    }
+});
+
+const hasOpenContainer = Symbol();
```

Comment:
How about adding a `description`?

```js
const hasOpenContainer = Symbol("hasOpenContainer");
```

Reply:
I guess thats not needed as we don't export it

Reply:
it's a unique symbol as default.
it would be better to have description for debugging.